lib/Importer/ONNX.cpp

```diff
@@ -80,6 +80,20 @@ size_t getPad(ArgumentDictionaryTy &dict) {
   return 0;
 }
 
+std::vector<size_t> getPads(ArgumentDictionaryTy &dict) {
+  if (dict.count("pads")) {
+    return getShape(dict["pads"]);
+  } else if (dict.count("auto_pad")) {
+    auto padStr = loadStr(dict["auto_pad"]);
+    if (padStr == "VALID")
```

Comment:
can you add {}. When there are more than a single line it's harder to read (comment in this case).